OpenRA.Mods.Common/Traits/Render/WithTurretAttackAnimation.cs

```diff
@@ -61,50 +51,45 @@ public WithTurretAttackAnimation(ActorInitializer init, WithTurretAttackAnimatio
 
 		void PlayAttackAnimation(Actor self)
 		{
-			if (!string.IsNullOrEmpty(info.AttackSequence))
-				wst.PlayCustomAnimation(self, info.AttackSequence);
+			if (!string.IsNullOrEmpty(Info.AttackSequence))
+				wst.PlayCustomAnimation(self, Info.AttackSequence);
 		}
 
 		void INotifyAttack.Attacking(Actor self, Target target, Armament a, Barrel barrel)
 		{
-			if (info.DelayRelativeTo == AttackDelayType.Attack)
```

Comment:
This appears to be almost completely duplicated with `INotifyAttack.PreparingAttack`.
Could you split out a `void NotifyAttack(Actor self, AttackDelayType type)` method that is called from each?

Reply:
Done.